### DISCUSSION TITLE
New idea

### DISCUSSION THREAD

**Discussion by thewh1teagle** [r0x0r/pywebview#811] [Ideas]
Firstly, 
That's AWESOME idea
That's the future!
Desktop apps using python & top web technologies!
I think that this idea missing single thing to make it ready for real world desktop apps solution
auto python runtime installer. like .NET apps have.
I thinking about make project exactly for that. 
The result - 
We may able to create single executable app in python and web frameworks, 
and "compile" it all to lightweight EXE ( less than 1MB )
with python "loader" that will download the correct python version if not exist yet, that will use virtual environment libraries (in some implemention)

**Comment by r0x0r** [comment]
While I welcome this idea, I am not eager to work on this solution. If somebody wants to give it a try, I will provide support on the pywebview side.

**Reply by thewh1teagle** [reply]
I'm eager to work on this solution
And i going to give it a try! 
If somebody can help me with designing and planning it would be great

**Comment by thewh1teagle** [comment]
https://github.com/thewh1teagle/dotpy
